ch_tools/chadmin/cli/zookeeper_group.py

```diff
@@ -266,3 +267,20 @@ def delete_ddl_task_command(ctx, tasks):
     """
     paths = [f"/clickhouse/task_queue/ddl/{task}" for task in tasks]
     delete_zk_nodes(ctx, paths)
+
+
+@zookeeper_group.command(name="clickhouse-hosts-cleanup")
+@option("-f", "--fqdn", type=str, help="Removed FQDNs, comma separated", required=True)
```

Comment:
Suggest to use `type=ListParamType()`. Then `fqdn.split(",")` wouldn't be required.

Reply:
Also consider replacing `@option` with `@argument` as it's required parameter.